README.md

```diff
@@ -13,68 +13,125 @@ npm install
 
 ## Usage
 
-Start the REPL from the terminal:
+#### Start the interactive shell
 ```bash
 node run
 ```
 The command prompt `KnowSheet> ` will appear.
 
-... or evaluate one expression from the command-line:
+#### Evaluate from the command-line
 ```bash
 echo 'HTTP(GET("http://httpbin.org/get?query=1")).body' | node run
 ```
+If `stdin` is not a `tty`, the shell starts in a non-interactive mode.
 
-... or evaluate and pass the result to another command:
+#### Evaluate and pass the result to another command
 ```bash
 echo 'HTTP(GET("http://httpbin.org/get?query=1")).body' | node run | cat
 ```
```

Comment:
Fixed in https://github.com/sompylasar/KnowSheet-REPL/commit/1abab3511848f6cd7c302855794fd0a2c0dd390c